src/drive/store/connectedRouter.js

```diff
@@ -13,10 +14,14 @@ const createParamsExtractor = routes => {
   return location => {
     const pathname = location.pathname
     for (let route of routes) {
-      const params = getParams(route, pathname)
-      if (params) {
-        return params
-      }
+      const match = matchPattern(route, pathname)
+      if (match && match.remainingPathname === '')
+        return fromPairs(
+          match.paramNames.map((paramName, index) => [
+            paramName,
+            match.paramValues[index]
+          ])
+        )
```

Comment:
I though react-router would use the first route that matches ? To have the same kind of behavior, shouldn't we use the same thing, and sort of routes so that the matching is the same as react-router ?

Reply:
From what I understand, it uses the first route that matches without any `remainingPathname`. `matchPattern ` seemed like the highest possible level I could borrow in terms of having a matching behavior, anything above that expects a full router configuration to work with.

Reply:
OK thanks